docs/tutorials/bson-v4.txt

```diff
@@ -485,36 +485,36 @@ strings to be used instead.
   stringified when used as hash keys:
 
   .. code-block:: ruby
-  
-  Hash.from_bson({foo: 'bar'}.to_bson)
-  # => {"foo"=>"bar"}
-  
-  Hash.from_bson({1 => 2}.to_bson)
-  # => {"1"=>2}
+
+    Hash.from_bson({foo: 'bar'}.to_bson)
+    # => {"foo"=>"bar"}
+
+    Hash.from_bson({1 => 2}.to_bson)
+    # => {"1"=>2}
 
 By default, the BSON library encodes ``Symbol`` hash values as strings and
 decodes BSON symbols into Ruby ``Symbol`` values:
 
   .. code-block:: ruby
 
-  {foo: :bar}.to_bson.to_s
-  # => "\x12\x00\x00\x00\x02foo\x00\x04\x00\x00\x00bar\x00\x00"
+    {foo: :bar}.to_bson.to_s
+    # => "\x12\x00\x00\x00\x02foo\x00\x04\x00\x00\x00bar\x00\x00"
 
-  # 0x02 is the string type
-  Hash.from_bson(BSON::ByteBuffer.new("\x12\x00\x00\x00\x02foo\x00\x04\x00\x00\x00bar\x00\x00".force_encoding('BINARY')))
-  # => {"foo"=>"bar"}
+    # 0x02 is the string type
+    Hash.from_bson(BSON::ByteBuffer.new("\x12\x00\x00\x00\x02foo\x00\x04\x00\x00\x00bar\x00\x00".force_encoding('BINARY')))
+    # => {"foo"=>"bar"}
 
-  # 0x0E is the symbol type
-  Hash.from_bson(BSON::ByteBuffer.new("\x12\x00\x00\x00\x0Efoo\x00\x04\x00\x00\x00bar\x00\x00".force_encoding('BINARY')))
-  # => {"foo"=>:bar}
+    # 0x0E is the symbol type
+    Hash.from_bson(BSON::ByteBuffer.new("\x12\x00\x00\x00\x0Efoo\x00\x04\x00\x00\x00bar\x00\x00".force_encoding('BINARY')))
+    # => {"foo"=>:bar}
```

Comment:
You didn't introduce this issue, but could you un-indent this entire block one level to remove the weird gray indentation line?